wordpress-mobile/WordPress-iOS
# Comments: Remove newCommentDetail feature flag

Fixes n/a

As titled, this removes the `newCommentDetail` feature flag. This also removes `CommentViewController` since it is only used to display the previous version of comment detail from My Sites. I've also noticed that `CommentViewController` is imported in `EditCommentViewController`, but it's never used.

## To Test

- Go to My Sites > Select any comment.
- Verify that the comment detail is displayed correctly.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested comment details in My Sites.

3. What automated tests I added (or what prevented me from doing so)
n/a 

PR submission checklist:
